inference-engine/samples/speech_sample/speech_sample.hpp

```diff
@@ -81,6 +81,16 @@ static const char context_window_message_r[] = "Optional. Number of frames for r
                                                "Works only with context window networks."
                                                " If you use the cw_r or cw_l flag, then batch size and nthreads arguments are ignored.";
 
+/// @brief message for output layers names
+static const char output_layers_names_massage[] = "Optional. Names of output layers for prepare output blobs. " \
+                                          "Names are specified with . " \
+                                          "Example: Relu_1.Relu_2. ";
+
+/// @brief message for output layers names
+static const char input_layers_names_massage[] = "Optional. Names of input layers for prepare input blobs. " \
+                                          "Names are specified with . " \
+                                          "Example: Relu_1.Relu_2. ";
```

Comment:
We should also add port number here. Also I suggest to use comma separated list instead of dots separated.

Reply:
"message", not "massage".
"Layer names for input blobs".
"The names", not "names".
"separated", not "specified". 
Why extra blank at the end?
"Relu" seems  very strange as an input layer type, I would use more neutral "input1" or "input2".

Do we really need separator at the end of the list?